consensus/dummy/README.md

```diff
@@ -4,30 +4,30 @@ Disclaimer: the consensus package in subnet-evm is a complete misnomer.
 
 The consensus package in go-ethereum handles block validation and specifically handles validating the PoW portion of consensus - thus the name.
 
-Since AvalancheGo handles consensus for Subnet-EVM, Subnet-EVM is just the VM, but we keep the consensus package in place to handle part of the block verification process.
+Since AvalancheGo handles consensus for subnet-evm, subnet-evm is just the VM, but we keep the consensus package in place to handle part of the block verification process.
```

Comment:
Was this caught by the linter, or just a one-off manual fix? In generaly, it's unclear to me where we use Subnet-EVM capitalization vs subnet-evm.